docs/application/toc_all.md

```diff
@@ -3,21 +3,27 @@
 ## [Overview](/application/index.md)
 
 ## Tizen Profiles
-
 ### [Overview](/application/profiles/index.md)
 ### [Wearable](/application/profiles/wearable.md)
 ### [Mobile](/application/profiles/mobile.md)
 ### [IoT](/application/profiles/iot.md)
 ### [TV](/application/profiles/tv.md)
 
-# .NET Application
 
+# .NET Application
 
 ## [Overview](/application/dotnet/index.md)
+
 ## [Learn about .NET](/application/dotnet/learn-about-dotnet.md)
 
-## Get Started
+<!--- vscode-ext/Tizen/dotnet.md and vstools/Tizen/dotnet.md should be merged into dotnet/get-started/first-app.md (PR#1599)
+## Get Started with Visual Studio Code Extension
+### [.NET application development](/application/vscode-ext/Tizen/dotnet.md)
 
+## Get Started with Visual Studio
+### [.NET application development](/application/vstools/Tizen/dotnet.md)
+--->
```

Comment:
@asm-foysal As you checked on https://github.com/Samsung/tizen-docs/pull/1599#issuecomment-1081330826,
this part should be changed to have similar form with Native and Web.